# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/metamail/35_mm-2.7-arghhh.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- mm2.7/src/bin/showexternal~	Mon Jun 22 10:55:36 1998
+++ mm2.7/src/bin/showexternal	Tue Jun 23 21:16:04 1998
@@ -215,7 +215,7 @@
 	    exit -1
 	endif
 	echo Subject: Automated Mail Server Request > "$NEWNAME"
-	echo To: ${server} >> "$NEWNAME"
+	echo To: "$server" >> "$NEWNAME"
 	echo "" >> "$NEWNAME"
 	sed -e 1,/^\$/d < $bodyfile >> "$NEWNAME"
 	echo "" >> "$NEWNAME"
